src/main/java/com/expensetracker/model/dto/YearExpensesDTO.java

```diff
@@ -0,0 +1,4 @@
+package com.expensetracker.model.dto;
+
+public record YearExpensesDTO(Object date, Object value, Object transaction) {
```

Comment:
Object? Wouldn't be better to use LocalDate?

Reply:
Spring has problems converting the month function to a LocalDate or a String.